repository/project.go

```diff
@@ -50,7 +48,7 @@ func (r *ProjectRepository) CreateProjectWithOwner(ctx context.Context, p svcmod
 		err = util.FinishTransaction(ctx, tx, err)
 	}()
 
-	_, err = r.dbpool.Exec(ctx, query.CreateProject, pgx.NamedArgs{
+	_, err = tx.Exec(ctx, query.CreateProject, pgx.NamedArgs{
```

Comment:
Just fix for refactor from previous PR. I forgot to actually use the created transaction :D